src/sentry/snuba/metrics/fields/base.py

```diff
@@ -235,11 +241,21 @@ class AliasedDerivedMetric(AliasedDerivedMetricDefinition, MetricObject):
     """
 
     def generate_metric_ids(self, projects: Sequence[Project]) -> Set[int]:
-        return {resolve_weak(org_id_from_projects(projects), self.raw_metric_mri)}
+        return {
+            resolve_weak(
+                UseCaseKey.RELEASE_HEALTH, org_id_from_projects(projects), self.raw_metric_mri
+            )
+        }
 
     def generate_filter_snql_conditions(self, org_id: int) -> Function:
         conditions = [
-            Function("equals", [Column("metric_id"), resolve_weak(org_id, self.raw_metric_mri)])
+            Function(
+                "equals",
+                [
+                    Column("metric_id"),
```

Comment:
same here, usecase need to be passed same way org_id is passed

Reply:
wdym ahmed?

Reply:
Cause we chatted earlier that there was a similar PR that he could look to for inspo which Meredith added with org_id so this a reference to that

Reply:
this fixed already here https://github.com/getsentry/sentry/blob/a2e7352dd2257e52cd3b5596d53f85932ef84ea3/src/sentry/snuba/metrics/fields/base.py#L252-L252